README.md

```diff
@@ -1 +1,51 @@
-# leaderboard
\ No newline at end of file
+# leaderboard
+Simple project that simply names with their scores, built with JavaScript
+
+### Built With
+- Html<br />
+- CSS
+- Javascript
+- webpack
+
+### Live Demo
+[live server](https://kusilaw.github.io/leaderboard/dist)
+
+### Getting Started 
+To get a local copy up and running follow these simple example steps
+- install dev dependencies
+  - npm start
+- Run sever
+  - npm run dev  
+- Update the dist folder 
+  - npm run build 
+
+#### Prerequisite:  
+  - Vscode editor 
+  - Webpack
+  - Knowldege About HTML / CSS Linters
+  - A web browser(chrome, etc)
+  - [Clone](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop ) the repository to your local machine
+  - Open index.html file with chrome
+
+
+### Author
+- Lawrence Addai Kusi
+  - Github: [@Githubhandle](https://github.com/kusiLaw)
+  - Twitter: [@Twitterhandle](https://twitter.com/kusilaw)
+  - Linkedin: [Linkedin](https://www.linkedin.com/in/lawrence-kusi-55a662104)
+
+
+### :handshake: Contributing
+Contributions, issues, and feature requests are welcome! 
+
+Feel free to check the [issues page](https://github.com/kusiLaw/portfolio/issues).
+
+### Show your support
+Give a star :star: if you like this project!
+
+
+### Acknowledgments
+Thanks to Microverse
+
+### 📝 License
+This Project is <a href ="https://opensource.org/licenses/MIT">MIT </a> licensed
```

Comment:
- [ ] I noticed that you have a license which is perfect :+1:  but the link to your license is not the correct one :cry:  Please add the correct link or path to your customized licence in order to make your readme file more professional :smile: